dff/utils/testing/cleanup_db.py

```diff
@@ -20,37 +26,67 @@
 
 
 async def delete_json(storage: JSONContextStorage):
+    """
+    Deletes all data from a JSON context storage.
```

Comment:
Use `Delete` instead of `Deletes` here and in other functions.
[PEP-257](https://peps.python.org/pep-0257/):
> The docstring is a phrase ending in a period. It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”.